pkg/services/publicdashboards/models/models.go

```diff
@@ -135,11 +113,17 @@ type SavePublicDashboardDTO struct {
 	PublicDashboard *PublicDashboard
 }
 
+type TimeRangeDTO struct {
+	From     string `json:"from"`
+	To       string `json:"to"`
+	Location string `json:"location"`
+}
+
 type PublicDashboardQueryDTO struct {
 	IntervalMs      int64
 	MaxDataPoints   int64
 	QueryCachingTTL int64
-	TimeRange       TimeSettings
+	TimeRange       TimeRangeDTO
```

Comment:
@evictorero wonder why Location is added to `TimeRangeDTO`? Think we should use standard `TimeRange` struct rather than introducing public dashboard specific DTO. What's the issue with placing the `Location` on `PublicDashboardQueryDTO`?

Reply:
You are talking about the `TimeSettings` struct that I am removing right? That is a model struct, we created it to support saving the `TimeSettings` data persisted for each public dashboard but we ended up not using it, there is a plan to remove it. On the other hand, we are also adding DTOs to separate the entities of the data transfer objects to remove coupling, that is why makes sense to have a DTO for the `TimeRange`.